Tests/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8-slim-buster
+
+WORKDIR /scs-compliance
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+ADD . .
```

Comment:
Do not use `ADD`. `ADD`'s usage is intended for URLs and Archive files. Rather use `COPY`, too.

Reply:
:+1: